server/src/com/thoughtworks/go/server/service/ArtifactsService.java

```diff
@@ -156,8 +161,14 @@ public void moveConsoleArtifacts(LocatableEntity locatableEntity) {
         }
     }
 
-    public File temporaryConsoleFile(JobIdentifier jobIdentifier) {
-        return chooser.temporaryConsoleFile(jobIdentifier);
+    public File temporaryConsoleFile(JobIdentifier jobIdentifier) throws IllegalArtifactLocationException {
```

Comment:
I am changing it to just consoleLogFile.
The service calls the chooser which branches out to temporaryConsole and findArtifact